config/routes.rb

```diff
@@ -4,7 +4,11 @@
   # Defines the root path route ("/")
   root 'users#index'
 
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+  resources :users, only: [:index, :show] do 
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:new, :create] 
```

Comment:
- [ ] Please make sure that you make use of these new and create routes. Currently, you are rendering and creating your comment at these routes which is wrong :blush: 
  ```ruby
     POST   /posts/:post_id/(.:format)     comments#create
   GET   /posts/:post_id/(.:format)     comments#new
  ```
  You are expected to render and create comments in these routes
  ```ruby
     POST   /posts/:post_id/comments(.:format)     comments#create
   GET   /posts/:post_id/comments/new(.:format)     comments#new
  ```